src/App.tsx

```diff
@@ -1,61 +1,122 @@
+import { useState } from 'react';
 import './App.scss';
+import { User } from './types/User';
+import { Todos } from './types/Todo';
+import { TodoList } from './components/TodoList';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+export function getUserById(userId: number): User | null {
+  return usersFromServer.find(user => user.id === userId) || null;
+}
+
+const initialTodos: Todos[] = todosFromServer.map(todo => ({
```

Comment:
you get initial todos from server, as for me preparedTodos suits better here
```suggestion
const preparedTodos: Todos[] = todosFromServer.map(todo => ({
```